### PR TITLE
feat: /data/spark/what-is-spark build

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Built page as per [doc](https://docs.google.com/document/d/1pyX-BOXmIZY6_8imIft1SAp1-S0i8bGYWEo5zoVEgQw/edit?tab=t.0) and [design](https://www.figma.com/design/uJUwbUAGDog4KCMivrMpnl/24.10-Data-Fabric?node-id=1498-8200&node-type=canvas&m=dev)
- Drive by:
  - Bumped Vanilla and Sass versions (sass bump needed for vanilla) 
  
## QA

- View the site locally in your web browser at: https://canonical-com-1409.demos.haus/data/spark/what-is-spark
- Check against doc and design 
- I qa'd the Vanilla bump and didn't notice anything broken, but please also have a look just in case

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-15058

### CODE REVIEW

**Comment by eliman11** [comment]
Thanks! A couple comments from me:
- [ ] Add alt text for Apache Spark logo in the hero
- [ ] Could you please add the legal copy at the bottom of the page? Sorry I forgot to paste it into the copydoc earlier!
- [ ] "Learn more about OpenSearch" changed into "Learn more about Spark" in the last section

**Comment by jmuzina** [comment]
We have found that the `.lazyloaded` div generated by the image template causes the aspect ratio to not be respected. In the short term this can be fixed by applying `display: contents` to the lazyloaded div:

```scss
  .p-image-container,
  [class^='p-image-container--'] {
    // If there is a child element that is not the image, don't let it affect the layout
    & > *:not(.p-image-container__image) {
      display: contents;
    }
}
```

This will be upstreamed to Vanilla shortly

**Comment by mtruj013** [comment]
@eliman11 about the alt text for the logo hero, can I ask why we're including it? We usually consider these decorative images and the r[ecommended practice](https://www.w3.org/WAI/tutorials/images/decorative/) is to leave an empty alt text for these so that screen readers will ignore it

**Comment by mtruj013** [comment]
@eliman11 @lyubomir-popov thanks both, I've adressed most of your comments if you could take another look?

Unfortunately the image container snippet is causing [some weirdness](https://www.loom.com/share/7bbafd1039644dd783be16edce961cf9?sid=a99efacc-f70f-4dd8-a1db-b5984f2474f3) with the image template. I've added the `p-image-container--3-2-on-medium` class to the three images regardless so that once the fix is on Vanilla it'll be automatically applied. (cc @jmuzina )

**Comment by jmuzina** [comment]
> Unfortunately the image container snippet is causing [some weirdness](https://www.loom.com/share/7bbafd1039644dd783be16edce961cf9?sid=a99efacc-f70f-4dd8-a1db-b5984f2474f3) with the image template. I've added the `p-image-container--3-2-on-medium` class to the three images regardless so that once the fix is on Vanilla it'll be automatically applied. (cc @jmuzina )

The fix for this has been merged to Vanilla, I'll be releasing it shortly as 4.18.1

**Comment by lyubomir-popov** [comment]
the mages still havew the 2-3 aspect ratio rather than 3-2 on medium/small 
image

**Comment by eliman11** [comment]
Thanks, looks good to me! Agreed about not including the alt text, thanks for pointing it out.

File: templates/data/spark/what-is-spark.html [hunk 1]
```diff
@@ -0,0 +1,468 @@
...
+            <h3 class="p-text--small-caps">Spark Rock container image</h3>
+          </div>
+          <div class="col-6 col-medium-3">
+            <p class="p-heading--5">Included in Ubuntu Pro</p>
+            <p>
+              Also included in Ubuntu Pro, you get support for Canonical's container image for Spark in GitHub Container Registry (GHCR) &ndash; the slimmest, fastest, niftiest Spark container image on the market, based on Ubuntu LTS. So solid and secure, we call it a Rock.
+            </p>
+            <hr class="p-rule--muted" />
+            <ul class="p-list--divided">
+              <li class="p-list__item is-ticked">Up to 10 years of Spark support per release track</li>
```
Comment (petesfrench) [inline templates/data/spark/what-is-spark.html:364]:
```suggestion
              <li class="p-list__item is-ticked">Up to 10 years of support per release track</li>
```

File: templates/data/spark/what-is-spark.html [hunk 2]
```diff
@@ -0,0 +1,468 @@
...
+            <p class="u-no-margin--bottom">
+              Building an effective, online data hub to facilitate access to enterprise data means ensuring solution scalability and reliability. Read the guide to gain insights into the value, use cases and challenges associated with building an enterprise data hub &ndash; whether on the public cloud or on-premise.
+            </p>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section--deep">
```
Comment (petesfrench) [inline templates/data/spark/what-is-spark.html:458]: High macro potential detected :eyes:

**Review by petesfrench — APPROVED** [review]
Looking good! Address the comments and add it to the sitemap and it is good to go

**Pull request merged**